src/main/java/org/casbin/adapter/HibernateAdapter.java

```diff
@@ -59,7 +59,7 @@ private void open() throws SQLException {
     private void createDatabase() {
         Session session = factory.openSession();
         Transaction tx = session.beginTransaction();
-        if (this.databaseProductName.contains("MySQL")) {
+        if (isSupportedDatabase(this.databaseProductName, "MySQL", "MariaDB")) {
```

Comment:
this.databaseProductName.contains("MySQL") || this.databaseProductName.contains("MariaDB")